CPRynner/CPRynner.py

```diff
@@ -220,7 +198,7 @@ def _get_username_and_password():
     else:
         username = ''
 
-    dialog = LoginDialog(username)
+    dialog = LoginDialog( username )
```

Comment:
No space after opening or before closing parenthesis is the accepted style in Python.